app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
```

Comment:
Code Efficiency: The `cp` command is not correctly checked. The command `cp` should be the first part of the split command, but you're only checking if it exists anywhere in the command. This might lead to incorrect results if the user inputs something like 'filename cp filename2'.